### ISSUE TITLE
Calculations in Facet filter do not add up

### ISSUE THREAD

**Issue by cncoleman** [humanitiesplusdesign/palladio-app#3]
Some problems I saw with the display [Facet Filter, with count by papers (letters)]:

1.  Ben Franklin appears as the Author of 1557 letters. That is correct. But he does not appear at all as recipient. That is incorrect. 
2. And if you add the "Name" dimension, it should have the sum of letters he authored and received but it only has 1557.
3. Another problem is the Gender dimension. It lists 1557 Male letters. But that is the total number of letters that Franklin wrote. What is it counting?

**Comment by esjewett** [comment]
Mostly resolved offline. We do need to do some work on the design attribute reporting like Gender, as what this is doing is listing the number of letters where a man was the author or recipient. This is still not immediately obvious using the application, and often the user will want to ask a more restrictive question about the gender distribution of just the authors or recipients. We need to have a way to do this that is easier than the current solution of unlinking/unextending dimensions.

**Comment by cncoleman** [comment]
I used the Palladio save file you sent yesterday evening. I opened it in the latest Palladio app from the repo. All looked good. I created another save file for count by letters. Then I went to the Data page and downloaded each of the tables. I opened another instance of Palladio in a new browser (because cache continues to be a problem for me when I re-open a save file). But when I tried to link the Papers table (correct 3443 rows) to the People table (774 rows) I had only 19 matches for Author. The Author dimension has been split on ":" and shows 628 values. There should be 628 matches from Author to the People table. I deleted the table and added it again. All good - 628 matches. Very strange.
